client/src/main/java/io/split/client/api/SplitView.java

```diff
@@ -24,13 +24,17 @@ public class SplitView {
     public List<String> treatments;
     public long changeNumber;
     public Map<String, String> configs;
+    public List<String> sets = new ArrayList<>();
 
     public static SplitView fromParsedSplit(ParsedSplit parsedSplit) {
         SplitView splitView = new SplitView();
         splitView.name = parsedSplit.feature();
         splitView.trafficType = parsedSplit.trafficTypeName();
         splitView.killed = parsedSplit.killed();
         splitView.changeNumber = parsedSplit.changeNumber();
+        if (parsedSplit.flagSets() != null) {
```

Comment:
correct me if I'm wrong but we will return empty not null

Reply:
Updated.